libs/managers/GitTools.cs

```diff
@@ -3,14 +3,40 @@
 using System;
 using System.Diagnostics;
 using System.IO;
-
+using System.Runtime.InteropServices;
+using Godot;
+using Godot.Collections;
 namespace GitTools
 {
-    public class GitRunner
+    class Util{
+        public static String GetGit()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                var output = new Godot.Collections.Array();
+                int exit_code = OS.Execute("where", new string[] { "git" }, true, output);
+                if (exit_code == 0)
+                {
+                    return (output[0] as string).StripEdges();
+                }
+            }
+            else// if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) //Which can be used on linux and mac?

```

Comment:
I understand that linux and mac use the same command? and this should work?

Reply:
This is correct, Windows = where, Linux and Mac OS = which.